src/exampleTypes/TermsStatsTable.js

```diff
@@ -33,7 +33,7 @@ let SelectSize = _.flow(
     options = [10, 25, 50, 100, 500, 1000],
     theme: { Select },
   }) => (
-    <Flex style={toolBarStyle}>
+    <Flex style={{ marginLeft: 12, ...toolBarStyle }}>
```

Comment:
this feels out of place given that it's a grey vest opinionated spacing 🤔

Reply:
Without this margin, we have no gap between SimpleFilter and SelectSize which looks broken. We have other margins and padding specified in pixels in this component